### PR TITLE
add Gathering information with "spacewalk-report"

### PR DESCRIPTION
**Pull request by keichwa**
from 2.1 docs.
https://github.com/SUSE/spacewalk/issues/6596

### CODE REVIEW

File: adoc/bp_chap_troubleshooting.adoc [hunk 1]
```diff
@@ -312,6 +312,72 @@ For more information see also:
...
+| System Groups Users | [command]``system-groups-users`` | Report of system groups users.
+| Installed Packages | [command]``system-packages-installed`` | List of packages installed on systems.
+| Users in the System | [command]``users`` | Lists all users registered to SUSE Manager.
+| Systems administered | [command]``users-systems`` | List of systems that individual users can administer.
+|===
+
+For more information about an individual report, run [command]``spacewalk-report`` with the option [option]``--info`` or [option]``--list-fields-info`` and the report name.
+The description and list of possible fields in the report will be shown.
+
+For further information, the [literal]``spacewalk-report(8)`` man page as well as the [option]``--help``parameter of the [command]``spacewalk-report`` program can be used to get additional information about the program invocations and their options.
```
Comment (Loquacity) [inline adoc/bp_chap_troubleshooting.adoc]: "For further information on program invocations and options, see ... "

File: adoc/bp_chap_troubleshooting.adoc [hunk 2]
```diff
@@ -312,6 +312,72 @@ For more information see also:
 ** https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=3444
 
 
+[[bp.chapt.suma3.troubleshooting.spacewalk-report]]
+== Gathering Information with [command]``spacewalk-report``
+
+There are instances where administrators may need a concise, formatted summary of their {susemgr} resources, whether it is to take inventory of their entitlements, subscribed systems, or users and organizations.
+Rather than gathering such information manually from the {susemgr} {webui}, {susemgr} includes the [command]``spacewalk-report`` command to fetch and display vital {susemgr} information at once.
```
Comment (Loquacity) [inline adoc/bp_chap_troubleshooting.adoc]: You've buried the lede ;) 
Try something like this:

`The [command]``spacewalk-report`` command is used to produce a variety of reports for system administrators.
These reports can be helpful for taking inventory of your entitlements, subscribed systems, users, and organizations.
Using reports is often simpler than gathering information manually from the {susemgr} {webui}, especially if you have many systems under management.`

Comment (keichwa) [reply adoc/bp_chap_troubleshooting.adoc]: Yes, I usually do ;)  In this case, it was literally copied from the 2.1 guide. I know this does not make it any better...

File: adoc/bp_chap_troubleshooting.adoc (hunk 1 above)
Comment (keichwa) [reply adoc/bp_chap_troubleshooting.adoc]: thanks! so much better

**Pull request merged**